pkg/collector/corechecks/snmp/internal/checkconfig/config_test.go

```diff
@@ -1072,7 +1072,7 @@ func Test_snmpConfig_setProfile(t *testing.T) {
 		},
 		Metrics: metrics,
 		MetricTags: []profiledefinition.MetricTagConfig{
-			{Tag: "interface", Column: profiledefinition.SymbolConfig{OID: "1.3.6.1.2.1.31.1.1.1.1", Name: "ifName"}},
+			{Tag: "location", Symbol: profiledefinition.SymbolConfigCompat{OID: "1.3.6.1.2.1.1.6.0", Name: "sysLocation"}},
```

Comment:
note for reviewers: I changed this symbol from ifName to sysLocation since it's more realistic to use a scalar OID here instead of a table OID